---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Exome sequencing and analysis of 454,787 UK Biobank participants
subtitle: ''
summary: ''
authors:
- Joshua D Backman
- Alexander H Li
- Anthony Marcketta
- Dylan Sun
- Joelle Mbatchou
- Michael D Kessler
- Christian Benner
- Daren Liu
- Adam E Locke
- Suganthi Balasubramanian
- Ashish Yadav
- Nilanjana Banerjee
- Christopher E Gillies
- Amy Damask
- Simon Liu
- Xiaodong Bai
- Alicia Hawes
- Evan Maxwell
- Lauren Gurski
- Kyoko Watanabe
- Jack A Kosmicki
- Veera Rajagopal
- Jason Mighty
- Marcus Jones
- Lyndon Mitnaul
- Eli Stahl
- Giovanni Coppola
- Eric Jorgenson
- Lukas Habegger
- William J Salerno
- Alan R Shuldiner
- Luca A Lotta
- John D Overton
- Michael N Cantor
- Jeffrey G Reid
- George Yancopoulos
- Hyun M Kang
- Jonathan Marchini
- Aris Baras
- Gonçalo R Abecasis
- Manuel AR Ferreira
tags: [population genetics, medical genetics, rare variants, whole exome sequencing, UK Biobank, genome-wide association, GWAS, exome-wide association, ExWAS]
categories: []
date: '2021-10-18'
lastmod: 2021-02-04T00:58:52-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-02-04T05:58:52.096526Z'
publication_types:
- '2'
abstract: 'A major goal in human genetics is to use natural variation to understand the phenotypic consequences of altering each protein-coding gene in the genome. Here we used exome sequencing1 to explore protein-altering variants and their consequences in 454,787 participants in the UK Biobank study2. We identified 12 million coding variants, including around 1 million loss-of-function and around 1.8 million deleterious missense variants. When these were tested for association with 3,994 health-related traits, we found 564 genes with trait associations at P ≤ 2.18 × 10−11. Rare variant associations were enriched in loci from genome-wide association studies (GWAS), but most (91%) were independent of common variant signals. We discovered several risk-increasing associations with traits related to liver disease, eye disease and cancer, among others, as well as risk-lowering associations for hypertension (SLC9A3R2), diabetes (MAP3K15, FAM234A) and asthma (SLC27A3). Six genes were associated with brain imaging phenotypes, including two involved in neural development (GBE1, PLD1). Of the signals available and powered for replication in an independent cohort, 81% were confirmed; furthermore, association signals were generally consistent across individuals of European, Asian and African ancestry. We illustrate the ability of exome sequencing to identify gene–trait associations, elucidate gene function and pinpoint effector genes that underlie GWAS signals at scale.'
publication: '[*Nature*](https://www.nature.com/articles/s41586-021-04103-z)'
doi: https://doi.org/10.1038/s41586-021-04103-z
---
