---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Matrix factorization and transfer learning uncover regulatory biology across
  multiple single-cell ATAC-seq data sets
subtitle: ''
summary: ''
authors:
- Rossin Erbe
- Michael D Kessler
- Alexander V Favorov
- Hariharan Easwaran
- Daria A Gaykalova
- Elana J Fertig
tags: []
categories: []
date: '2020-05-11'
lastmod: 2021-02-04T00:58:53-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-02-04T05:58:52.926305Z'
publication_types:
- '2'
abstract: 'While the methods available for single-cell ATAC-seq analysis are well optimized for clustering cell types, the question of how to integrate multiple scATAC-seq data sets and/or sequencing modalities is still open. We present an analysis framework that enables such integration across scATAC-seq data sets by applying the CoGAPS Matrix Factorization algorithm and the projectR transfer learning program to identify common regulatory patterns across scATAC-seq data sets. We additionally integrate our analysis with scRNA-seq data to identify orthogonal evidence for transcriptional regulators predicted by scATAC-seq analysis. Using publicly available scATAC-seq data, we find patterns that accurately characterize cell types both within and across data sets. Furthermore, we demonstrate that these patterns are both consistent with current biological understanding and reflective of novel regulatory biology.'
publication: '[*Nucleic acids research*](https://academic.oup.com/nar/article/48/12/e68/5835820?login=true)'
doi: https://doi.org/10.1093/nar/gkaa349
---
