---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program
subtitle: ''
summary: ''
authors:
- Daniel Taliun
- Daniel N Harris
- Michael D Kessler
- Jedidiah Carlson
- Zachary A Szpiech
- Raul Torres
- Sarah A Gagliano Taliun
- André Corvelo
- Stephanie M Gogarten
- Hyun Min Kang
- \.\.\.
- NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium
- George J. Papanicolaou
- Deborah A. Nickerson
- Sharon R. Browning
- Michael C. Zody
- Sebastian Zöllner
- James G. Wilson
- L. Adrienne Cupples
- Cathy C. Laurie
- Cashell E. Jaquish
- Ryan D. Hernandez
- Timothy D. O’Connor
- Gonçalo R. Abecasis
tags: [population genetics, rare variants, imputation, topmed, nhlbi, whole genome sequencing]
categories: []
date: '2021-02-10'
lastmod: 2021-02-04T00:58:52-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-02-04T05:58:52.096526Z'
publication_types:
- '2'
abstract: 'The Trans-Omics for Precision Medicine (TOPMed) programme seeks to elucidate the genetic architecture and biology of heart, lung, blood and sleep disorders, with the ultimate goal of improving diagnosis, treatment and prevention of these diseases. The initial phases of the programme focused on whole-genome sequencing of individuals with rich phenotypic data and diverse backgrounds. Here we describe the TOPMed goals and design as well as the available resources and early insights obtained from the sequence data. The resources include a variant browser, a genotype imputation server, and genomic and phenotypic data that are available through dbGaP (Database of Genotypes and Phenotypes)1. In the first 53,831 TOPMed samples, we detected more than 400 million single-nucleotide and insertion or deletion variants after alignment with the reference genome. Additional previously undescribed variants were detected through assembly of unmapped reads and customized analysis in highly variable loci. Among the more than 400 million detected variants, 97% have frequencies of less than 1% and 46% are singletons that are present in only one individual (53% among unrelated individuals). These rare variants provide insights into mutational processes and recent human evolutionary history. The extensive catalogue of genetic variation in TOPMed studies provides unique opportunities for exploring the contributions of rare and noncoding sequence variants to phenotypic variation. Furthermore, combining TOPMed haplotypes with modern imputation methods improves the power and reach of genome-wide association studies to include variants down to a frequency of approximately 0.01%.'
publication: '[*Nature*](https://www.nature.com/articles/s41586-021-03205-y)'
doi: https://doi.org/10.1038/s41586-021-03205-y
---
