---
# Documentation: https://wowchemy.com/docs/managing-content/

title: A reference methylome database and analysis pipeline to facilitate integrative
  and comparative epigenomics
subtitle: ''
summary: ''
authors:
- Qiang Song
- Benjamin Decato
- Elizabeth E Hong
- Meng Zhou
- Fang Fang
- Jianghan Qu
- Tyler Garvin
- Michael Kessler
- Jun Zhou
- Andrew D Smith
tags: [DNA sequencing, methylation, epigenomics, bisulfite sequencing, genetics]
categories: []
date: '2013-12-06'
lastmod: 2021-02-04T00:58:50-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-02-04T05:58:50.073124Z'
publication_types:
- '2'
abstract: 'DNA methylation is implicated in a surprising diversity of regulatory, evolutionary processes and diseases in eukaryotes. The introduction of whole-genome bisulfite sequencing has enabled the study of DNA methylation at a single-base resolution, revealing many new aspects of DNA methylation and highlighting the usefulness of methylome data in understanding a variety of genomic phenomena. As the number of publicly available whole-genome bisulfite sequencing studies reaches into the hundreds, reliable and convenient tools for comparing and analyzing methylomes become increasingly important. We present MethPipe, a pipeline for both low and high-level methylome analysis, and MethBase, an accompanying database of annotated methylomes from the public domain. Together these resources enable researchers to extract interesting features from methylomes and compare them with those identified in public methylomes in our database.'
publication: '[*PloS one*](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0081148)'
doi: https://doi.org/10.1371/journal.pone.0081148
---
