---
# Documentation: https://wowchemy.com/docs/managing-content/

title: De novo mutations across 1,465 diverse genomes reveal mutational insights and
  reductions in the Amish founder population
subtitle: ''
summary: ''
authors:
- Michael D. Kessler
- Douglas P. Loesch
- James A. Perry
- Nancy L. Heard-Costa
- Daniel Taliun
- Brian E. Cade
- Heming Wang
- Michelle Daya
- John Ziniti
- Soma Datta
- Juan C. Celedón
- Manuel E. Soto-Quiros
- Lydiana Avila
- Scott T. Weiss
- Kathleen Barnes
- Susan S. Redline
- Ramachandran S. Vasan
- Andrew D. Johnson
- Rasika A. Mathias
- Ryan Hernandez
- James G. Wilson
- Deborah A. Nickerson
- Goncalo Abecasis
- Sharon R. Browning
- Sebastian Zöllner
- Jeffrey R. O’Connell
- Braxton D. Mitchell
- Timothy D. O’Connor
tags: [de novo mutation, DNMs, mutation rate, genetics, genetic ancestry, Amish]
categories: []
date: '2020-02-04'
lastmod: 2021-02-04T00:58:53-05:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-02-04T05:58:53.426927Z'
publication_types:
- '2'
abstract: 'De novo mutations (DNMs), or mutations that appear in an individual despite not being seen in their parents, are an important source of genetic variation whose impact is relevant to studies of human evolution, genetics, and disease. Utilizing high-coverage whole-genome sequencing data as part of the Trans-Omics for Precision Medicine (TOPMed) Program, we called 93,325 single-nucleotide DNMs across 1,465 trios from an array of diverse human populations, and used them to directly estimate and analyze DNM counts, rates, and spectra. We find a significant positive correlation between local recombination rate and local DNM rate, and that DNM rate explains a substantial portion (8.98 to 34.92%, depending on the model) of the genome-wide variation in population-level genetic variation from 41K unrelated TOPMed samples. Genome-wide heterozygosity does correlate with DNM rate, but only explains <1% of variation. While we are underpowered to see small differences, we do not find significant differences in DNM rate between individuals of European, African, and Latino ancestry, nor across ancestrally distinct segments within admixed individuals. However, we did find significantly fewer DNMs in Amish individuals, even when compared with other Europeans, and even after accounting for parental age and sequencing center. Specifically, we found significant reductions in the number of C→A and T→C mutations in the Amish, which seem to underpin their overall reduction in DNMs. Finally, we calculated near-zero estimates of narrow sense heritability (h2), which suggest that variation in DNM rate is significantly shaped by nonadditive genetic effects and the environment.'
publication: '[*Proceedings of the National Academy of Sciences*](https://www.pnas.org/content/117/5/2560.short)'
#url_pdf: http://www.pnas.org/content/early/2020/01/17/1902766117.abstract https://www.pnas.org/content/pnas/early/2020/01/17/1902766117.full.pdf
doi: 10.1073/pnas.1902766117
---
